homeassistant/components/geniushub/climate.py

```diff
@@ -0,0 +1,204 @@
+"""
+Supports Genius hub to provide climate controls.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/climate.geniushub/
+"""
+import logging
+
+from homeassistant.components.climate import ClimateDevice
+from homeassistant.components.climate.const import (
+    STATE_ECO, STATE_HEAT, STATE_AUTO, STATE_IDLE,
+    SUPPORT_TARGET_TEMPERATURE, SUPPORT_OPERATION_MODE,
+    SUPPORT_ON_OFF, SUPPORT_AWAY_MODE)
+from homeassistant.components.geniushub import GENIUS_HUB
```

Comment:
Imports from the component package should be relative. This will allow custom component override.
```py
from . import GENIUS_HUB
```